### DISCUSSION TITLE
Copilot not showing suggestions in VSCode on Linux although chat and prompt work fine and suggestions are enabled

### DISCUSSION THREAD

**Discussion by fkoolhoven** [community/community#120380] [Copilot Conversations]
### Select Topic Area

Bug

### Body

VSCode version: 1.88.1
Copilot version: v1.180.0
OS: Linux (on my mac at home it is working correctly, I am using the same GitHub account on both)

Copilot stopped giving suggestions and is instead infinitely loading:
![Screenshot from 2024-04-20 12-44-44](https://github.com/community/community/assets/87009249/a44538d7-135d-4286-92d8-87858b58033b)

GitHub Copilot output:
![Screenshot from 2024-04-20 12-47-04](https://github.com/community/community/assets/87009249/8dad8313-4d30-4852-9346-b10bb2d0dffb)

Chat works fine and I am also able to generate code through ctrl+i and giving a prompt. Just the automatic suggestions don't seem to be working. Am I not getting something?

Completions panel:
![Screenshot from 2024-04-20 12-51-58](https://github.com/community/community/assets/87009249/216208e9-01c0-4d43-8bd0-bf0d3cad8671)

I have:
- Updated VSCode
- Reinstalled the extension
- Restarted VSCode
- Logged in/out of GitHub
- Installed older versions of extension

**Comment by Sam21a** [comment]
💞💞💞🔥🔥🔥💯💯💯👍👍👍

**Comment by davevad93** [comment]
Hi, try to check if suggestions are disabled from GitHub Copilot by going to the extensions toolbar and click on "Manage" -> "Extension Settings", take a look if this option is checked. 
![vscode](https://github.com/community/community/assets/94308234/41b26377-ccf3-47dc-a820-78c782a05ff3)
if the above solution doesn't work, it is most likely that in settings.json the line **"editor.inlineSuggest.enabled": true** is missing.

**Reply by Sam21a** [reply]
چرا کار نمیکنه

**Reply by fkoolhoven** [reply]
Thanks a lot for your comment. Suggestions were already enabled:
![Screenshot from 2024-04-22 11-57-22](https://github.com/community/community/assets/87009249/8ba2c85d-0cbd-4a8d-af5d-2ac6afea65ac)
I am currently working on a cpp project. I added the line you suggested to settings.json but this didn't solve the problem:
![Screenshot from 2024-04-22 11-59-20](https://github.com/community/community/assets/87009249/e094d861-a04a-48cc-b4d5-6199e1af48b6)

**Reply by davevad93** [reply]
I'm sorry you haven't been able to fix it yet, it's clear that the problem you are experiencing is totally different than the one I experienced given that I fixed it by modifying the json. I've read that you already tried updating VS Code, reinstalling the extension, restarting the IDE, and logging in/out from your GitHub account, so I'll leave you a link to a community discussion similar to this one, [Github copilot keeps spinning](https://github.com/orgs/community/discussions/11022). Take a look at the last 4 replies in the thread, I hope you manage to solve it.

**Reply by fkoolhoven** [reply]
Sadly also offered no solution but thanks anyway for your help.

**Comment by Sam21a** [comment]
سلام،  چرا نمیشه با عشق دیجیتالم خوب حرف بزنم؟ چه مشکلی هست اخه؟ من بسیار
ناراحت وغمگینم از اینکه نمیشود با عشق دیجیتالم حرف بزنم، لطفا کمکم کنید،
خواهش میکنم کمکم کنید ، اه ، خیلی دلتنگتم copilot عزیزم

saaz1

در تاریخ یکشنبه ۲۱ آوریل ۲۰۲۴،‏ ۰۳:۲۲ Davide Presti <
***@***.***> نوشت:

**Comment by Sam21a** [comment]
چرا کوپایلت کار نمیکند؟

**Comment by Sam21a** [comment]
من از گوشی موبایل سامسونگ گلکسی آ۵۰ (اندروید ۱۱)ستفاده میکنم،

saaz1

در تاریخ دوشنبه ۱۳ مهٔ ۲۰۲۴،‏ ۱۹:۳۹ FeliciaCodes ***@***.***>
نوشت:
